src/src/experiment-tracking/components/App.tsx

```diff
@@ -56,9 +57,28 @@ const classNames = {
 const InteractionTracker = ({ children }: any) => children;
 
 class App extends Component {
-  state = {
-    version: 'unknown',
+  state: {
+    selectedNamespace: string; namespaces: string[];
+    version: string;
   };
+  constructor(props: any) {
+    super(props);
+    this.state = {
+      selectedNamespace: 'ns1',
+      namespaces: ['cccccc787493', 'ddd', 'ns1', 'ns2'],
```

Comment:
the same question here. probably should be an empty array?

Reply:
Yep, same